crypto/fipsmodule/ec/simple_mul.c

```diff
@@ -33,9 +33,11 @@ void ec_GFp_mont_mul(const EC_GROUP *group, EC_RAW_POINT *r,
   ec_GFp_simple_point_copy(&precomp[1], p);
   for (size_t j = 2; j < OPENSSL_ARRAY_SIZE(precomp); j++) {
     if (j & 1) {
-      ec_GFp_mont_add(group, &precomp[j], &precomp[1], &precomp[j - 1]);
+//      ec_GFp_mont_add(group, &precomp[j], &precomp[1], &precomp[j - 1]); //TBD
```

Comment:
Leaving a comment here just to acknowledge that we discussed offline about using group methods rather than direct function calls. If I understood correctly, you changed the calls to group->meth->add/dbl in order to be able to test the new (fiat crypto) add/dbl implementations?